test_util/src/lib.rs

```diff
@@ -191,16 +191,16 @@ pub fn run_python_script(script: &str) {
 }
 
 #[derive(Debug, Default)]
-pub struct CheckOutputIntegrationTest {
-  pub args: &'static str,
-  pub output: &'static str,
-  pub input: Option<&'static str>,
-  pub output_str: Option<&'static str>,
+pub struct CheckOutputIntegrationTest<'a> {
+  pub args: &'a str,
+  pub output: &'a str,
+  pub input: Option<&'a str>,
+  pub output_str: Option<&'a str>,
```

Comment:
Please revert, unrelated change

Reply:
It is related to the commit already requested to be reverted ;)